src/classes/LineItemTriggerHandler.cls

```diff
@@ -17,6 +17,7 @@ public with sharing class LineItemTriggerHandler {
 		// load required data: 1) related orders 2) force reload all line items for those orders
 		List<Order__c> orders = [SELECT Purchased__c,Product_Category_1__c,Product_Category_Units_1__c,Product_Category_2__c,Product_Category_Units_2__c FROM Order__c WHERE Id in :orderIds];
 		List<Line_Item__c> lineItems = [SELECT Order__c, Amount__c, Lot__r.Product_Category__c, Units__c FROM Line_Item__c WHERE Order__c in :orderIds];
```

Comment:
`orders2` does not match naming conventions. Please rename (hint: `myVariable`).